src/rules/comment-empty-line-before/__tests__/index.js

```diff
@@ -17,12 +17,16 @@ function alwaysTests(tr) {
   tr.ok("a {}\r\n\r\n/** comment */", "CRLF")
   tr.ok("a { color: pink;\n\n/** comment */\ntop: 0; }")
 
-  tr.notOk("/** comment */\n/** comment */", messages.expected)
-  tr.notOk("/** comment */\r\n/** comment */", messages.expected, "CRLF")
   tr.notOk("a { color: pink;\n/** comment */\ntop: 0; }", messages.expected)
   tr.notOk("a { color: pink;\r\n/** comment */\r\ntop: 0; }", messages.expected, "CRLF")
 }
 
+function alwaysTests(tr) {
+  alwaysTestsCommon(tr)
+  tr.notOk("/** comment */\n/** comment */", messages.expected)
+  tr.notOk("/** comment */\r\n/** comment */", messages.expected, "CRLF")
+}
+
 testRule("always", tr => {
   alwaysTests(tr)
 
```

Comment:
I'd suggest not creating a new `alwaysTestsCommon()` function, but instead simply removing the no-longer-common tests from it and adding what's needed elsewhere. So the test for "always" with no options would perform `alwaysTests(tr)` and also perform those two tests about newlines between comments. Make sense? (I just think the `alwaysTests()` thing is already kind of unpleasant — I do it when I'm desperate not to handwrite more tests — and I don't want to compound it.)